src/main/scala/io/iohk/ethereum/vm/WorldStateProxy.scala

```diff
@@ -29,6 +29,7 @@ trait WorldStateProxy[WS <: WorldStateProxy[WS, S], S <: Storage[S]] {
 
   def getCode(address: Address): ByteString
   def getStorage(address: Address): S
+  def getBlockHeader(number: BigInt): Option[BlockHeader]
```

Comment:
@AlanVerbner are you aware of this. Will that fit in the implementation of WPS?

Also @adamsmo shouldn't we simply return the hash? We don't have a use for the whole `BlockHeader`.